lib/commands/update.js

```diff
@@ -179,6 +181,25 @@ class UpdateCommand extends Command {
         });
     }
 
+    rollbackFromFail(error, argv) {
+        const previousVersion = this.system.getInstance().cliConfig.get('previous-version');
+        this.ui.error(error);
```

Comment:
What is the output format of 

> this.ui.error(error);

?

Reply:
I'm not 100% sure 😬 I think the error would be a ListrError so it would be something like
```
One or more errors occurred.

1 Restarting Ghost

try running ghost doctor...
{docs help}
```

I figured it would be nice to use our native error recording functionality, but if it leaves the user confused that might not be the best idea 🤔

/cc @acburdine